schema/datatypes/hardware_info.fbs

```diff
@@ -52,6 +52,8 @@ table HardwareInfo {
     ip_address: solarxr_protocol.datatypes.Ipv4Address;
 
     board_type: string;
+
+    hardware_identifier: string;
```

Comment:
Add a comment explaining what this is, its not immediately clear to me.

Reply:
a unique identifier for the device, Eiren asked for it to be an string for flexibility's sake

Reply:
maybe its a mac address, or the usb port it is in, or maybe it is an ip. It's mainly for the user's sake

Reply:
Commenting in the PR is less helpful than adding the comment to the schema :)
Please add it with `///` so that others can see in the generated code.